sdks/python/apache_beam/testing/benchmarks/nexmark/nexmark_util.py

```diff
@@ -0,0 +1,85 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Utilities for the Nexmark suite.
+
+The Nexmark suite is a series of queries (streaming pipelines) performed
+on a simulation of auction events. This util includes a Command class used
+to terminate the streaming jobs launched in nexmark_launcher.py.
+
+Usage:
+To run a process for a certain duration, define in the code:
+  command = Command(process_to_terminate, args)
+  command.run(timeout=duration)
+
+To test, run in a shell:
+  $ python nexmark_util.py
+"""
+from __future__ import print_function
+
+import logging
+import threading
+import time
+from multiprocessing import Process
+
+
+logging.basicConfig(level=logging.DEBUG,
```

Comment:
Please default to INFO logging level. (Similar to other places.)

Reply:
I added a --loglevel option.